### ISSUE TITLE
Screen Reader is not Announcing Full and Proper info About High Charts.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#17363]
#### Expected behaviour
Screen reader should announce such as - Highcharts interactive chart. region interactive graphic chart created with  high charts with Chart Month on x – axis and Average Rainfall(mm) on y -axis. Bar series 1 of 4 with 12 bars. Region 1. Jan 49.9 Tokyo , bar graph.

#### Actual behaviour:
1) Screen reader is not announcing the information about x-axis and y-axis.
2) Screen reader also announces extra info which is not visible on screen such as - Created with High chart 10.0.0
3) Screen reader announcing as- Bar series 1 of 4 with 12 bars. Region 1. Jan 49.9 Tokyo , bar graph.

#### Steps to reproduce

1.Open URL: https://www.highcharts.com/demo/column-basic in the Edge browser
2.Navigate to ‘Basic Column’ under ‘Column and Bar Charts’ in the left navigation.
3. Enable NVDA.
4. Navigate to Bar-graph and observe whether NVDA announces descriptive information about x-axis and y-axis

#### Product version

OS: Windows 11 22H2 (OS BUILD 25131.1000)
Edge browser: 102.0.1245.33

Screenshot:
![image](https://user-images.githubusercontent.com/93735775/172267476-abf67870-ea1e-44e7-a552-b9d0b6ea659e.png)

**Comment by msftedad** [comment]
@oysteinmoseng 
Expected result is Screen reader should announce **Month on x – axis and Average Rainfall(mm) on y -axis** including Bar series 1 of 4 with 12 bars. Region 1. Jan 49.9 Tokyo, bar graph.
but actual result is screen reader is just announcing Bar series 1 of 4 with 12 bars. Region 1. Jan 49.9 Tokyo, bar graph.

### Please refer to the video attachment
https://drive.google.com/file/d/1pQAGt5BHHpfURde9lV2sB_wctgX6aUKL/view?usp=sharing

**Comment by oysteinmoseng** [comment]
@msftedad After more testing with NVDA and Edge, it seems to be working as expected. It will read the x and y-axis information, but before the elements in your video. If you navigate with up/down arrow from the beginning of the region it should read them out. NVDA is reading out more elements than needed, and repeating some information, but this is an NVDA issue and not something that we can change without breaking support for other screen readers.

Note that you can also change the configuration of Highcharts if you want specific items to read differently, see https://www.highcharts.com/docs/accessibility/configure-the-accessibility-module. 

I am closing this for now, but feel free to comment to reopen if I have misunderstood anything.
